tests/ShareUrl/DynamicScenarioBuilderTest.php

```diff
@@ -67,6 +74,7 @@ public function testBuild()
                 'wanted_auth_types' => [],
                 'wanted_remember_me' => false,
                 'wanted_remember_me_optional' => false,
+                'identity_profile_requirements' => null
```

Comment:
I think it's best to be explicit here, and include this null value (like you've done), rather than omitting it. @BenSmithYoti I think this matches the .NET implementation, right?

Reply:
Another e.g. is `issuing_authority_sub_check ` on `RequestedDocumentAuthenticityConfig`  

In short I believe the implementation in this PR will send up the following JSON: `"identity_profile_requirements" : null,` and I believe this will be handled fine by the API (same behaviour as if identity_profile_requirements was excluded from the JSON).